frontend/dashboard/pages/CreateService/CreateService.tsx

```diff
@@ -17,7 +17,7 @@ import { ServerCodes } from 'app-shared/enums/ServerCodes';
 import { useCreateAppFormValidation } from './hooks/useCreateAppFormValidation';
 import { navigateToAppDevelopment } from './utils/navigationUtils';
 
-const DASHBOARD_ROOT_ROUTE: string = '/';
+export const DASHBOARD_ROOT_ROUTE: string = '/';
```

Comment:
Consider moving this into `shared/src/constants.js`, alternatively just using `STUDIO_ROOT_BASENAME` from constants that has the same value 😊

Reply:
I moved `DASHBOARD_ROOT_ROUTE` into `shared/src/constants.js` as you suggested. I couldn't use `STUDIO_ROOT_BASENAME` as it does not seem to be used for the same root (e.g. in the `/dashboard/` route, the first `/` is `STUDIO_ROOT_BASENAME` and the second `/` is `DASHBOARD_ROOT_ROUTE`).

Based on your suggestion, I also created a new issue to centralize all routes to prevent duplicating them: https://github.com/Altinn/altinn-studio/issues/12624